doc/doc-site/docs/off-chain/dss/changelog.md

```diff
@@ -13,11 +13,18 @@ Stable DSS builds are tracked on the [`dss-stable`] branch with tags like [`dss-
 1. Merge `main` into `dss-stable`.
 1. Push annotated tag to head of `dss-stable`.
 
-## [v2.1.0] (in progress)
+## [v2.1.0-rc1]
 
 ### Added
 
 - Price level events over MQTT (disabled by default, enable by adding `MQTT_PRICE_LEVELS=yes`, see `src/docker/example.env`) ([#753]).
+- Fix the `/tickers` `base_volume_nominal` field ([#749]).
+- Optimize the `/tickers` endpoint ([#731]).
+- Add suspend and resume functionality for the DSS when deployed on GCP ([#736]).
+- Add more details in aggregator logging ([#738]).
+- Fix a bug that would corrupt your balance if your market account handle started with `0x0` ([#732]).
+- Add the `all` liquidity group by default to all markets ([#728]).
+- Remove the market registration pipeline (counted markets registered per day) ([#727]).
```

Comment:
Where is #744 ?

Reply:
And #739?

Reply:
And #737?

Reply:
And #730?

Reply:
#729

Reply:
#725

Reply:
Also -> `Remove the unused market registration pipeline, a development stub that counted markets registered per day`

Reply:
The `tokio` and workspace changes are a result of incorporating the new upstream changes to the processor (https://github.com/econia-labs/aptos-indexer-processors/pull/24, which wasn't originally included in this PR)

These do represent a diff in DSS source code, so how about combining all of them into one?

As for removing dead code I think it is also worth mentioning since it too constitutes a diff in DSS source

Result would be a new "changes" section

```
## Changes

- Refactor layout for upstream processor changes ([#725], [#744], [Processor #24])
- Remove dead code ([#737])
```

Reply:
For me, the purpose of the changelog is not to document work done but to inform downstream users of what has changed that could impact them. Us removing dead code or using a newer version of the processor does not impact them IMO. If you think we should rather see it as a documentation of the work that has been done, please let me know, but I don't really see the point in that. I think it is more useful for all DSS users to have a clear list of things that impact them rather than them having to parse through all the changes for potentially interesting things.